README.md

```diff
@@ -63,4 +75,27 @@ Here with the plot-{id} (where id is the same as the x-plot: {id} in the field y
 - Specify the parameters for each channel and have the options for them to be randomly generated.
 - Specify the payload you want to send.
 
+## Contributing
+
+Read [CONTRIBUTING](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md) guide.
+
+## Contributors ✨
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/NektariosFifes"><img src="https://avatars.githubusercontent.com/u/61620751?v=4?s=100" width="100px;" alt=""/><br /><sub><b>NektariosFifes</b></sub></a><br /><a href="https://github.com/asyncapi/fluffy-robot/commits?author=NektariosFifes" title="Code">💻</a> <a href="https://github.com/asyncapi/fluffy-robot/commits?author=NektariosFifes" title="Documentation">📖</a> <a href="#ideas-NektariosFifes" title="Ideas, Planning, & Feedback">🤔</a> <a href="#maintenance-NektariosFifes" title="Maintenance">🚧</a> <a href="https://github.com/asyncapi/fluffy-robot/commits?author=NektariosFifes" title="Tests">⚠️</a></td>
```

Comment:
In the ideas section could we add the related issues opened by each person?
https://github.com/asyncapi/simulator/issues?q=author%3ANektariosFifes

Reply:
We cannot change this content unfortunately as it is autogenerated 🙂